Approximate_convex_decomposition/include/CGAL/approx_decomposition.h

```diff
@@ -0,0 +1,87 @@
+#ifndef CGAL_APPROXIMATE_CONVEX_DECOMPOSITION_APPROX_DECOMPOSITION_H
+#define CGAL_APPROXIMATE_CONVEX_DECOMPOSITION_APPROX_DECOMPOSITION_H
+
+#include <CGAL/Kernel_traits.h>
+#include <CGAL/boost/graph/helpers.h>
+#include <CGAL/internal/Approximate_convex_decomposition/approx_decomposition.h>
+#include <CGAL/internal/Approximate_convex_decomposition/concavity.h>
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
```

Comment:
you should not include the kernel. If you want to have a default, you can use `Kernel_traits` to use the kernel of the points associated with the vertex point map.

Reply:
But there is no vertex point map.

Reply:
There should be :)
I suggest to look at a function from polygon mesh processing that is using named parameters to see how you can do it. For example you can write something like that:
```
template <class TriangleMesh , class NamedParameters>
void toto( const TriangleMesh& pmesh
              , const NamedParameters& np)
{
  using boost::choose_param;
  using boost::get_param;
  
  typename GetGeomTraits<PolygonMesh, NamedParameters>::type Traits;
  typename GetVertexPointMap<PolygonMesh, NamedParameters>::const_type
    vpm = choose_param(get_param(np, internal_np::vertex_point),
                       get_const_property_map(CGAL::vertex_point, pmesh));
}
```